doc/security/specs/device_life_cycle/_index.md

```diff
@@ -0,0 +1,646 @@
+---
+title: "Device Life Cycle"
+---
+
+<p style="text-align: right">
+    Version: 0.1</p>
+
+<p style="text-align: right">
+  Contributors:
+  <a href="http://github.com/tjaychen">@tjaychen</a>,
+  <a href="http://github.com/moidx">@moidx</a>,
+  <a href="http://github.com/ccgori">@ccgori</a>
+</p>
+
+## Overview
+
+This document describes the OpenTitan device operational states supported for
+manufacturing, provisioning, regular operation and RMA[^1].
+
+Provisioning refers to the process in which a device creates a unique and
+trusted identity.
+
+Manufacturing refers to the process in which a device is tested and prepared for
+silicon correctness prior to production.
+
+## Background
+
+First see [here]({{< relref "doc/security/logical_security_model" >}}) for
+OpenTitan's logical entity breakdowns.
+
+OpenTitan implements standard device life cycle manufacturing states which are
+configured via OTP[^2] memory. These states allow the **silicon creator** to
+manage the state of the device as it is being manufactured and provisioned for
+shipment.
+
+An additional set of life cycle states are used to encode the device ownership
+state. A device that has been personalized with a unique Silicon Creator
+Identity, can be provisioned with a Silicon Owner Identity and Credentials. This
+enables the Silicon Owner to run signed code on the device. The process of
+assigning a Silicon Owner to a device is referred to as Ownership Transfer.
+Depending on the product SKU configuration, a device may support a single and
+permanent ownership transfer or multiple ones. The multiple ownership transfer
+use case has additional state control requirements which are described in more
+detail in later sections.
+
+## Architecture
+
+Life cycle states are defined by a combination of OTP and flash. OTP state
+transitions cannot be reverted, while flash defined states are meant to be
+'flexible', especially when ownership transfer is required. This implies that
+when OTP states are transitioned from A to B, we cannot go back to A from B.
+However, it is still possible to transition from B to C, assuming such a
+transition is legal and allowed by the life cycle definition.
+
+See the table below for more details.
+
+<img src="device_life_cycle_fig1.svg" alt="Fig1" style="width: 800px;"/>
+
```

Comment:
I noticed that the figure shows all of the states mentioned in the tables below except for the INVALID state. Should the INVALID state be added to this diagram / similar to the SCRAP state shown? Entry ARC might be something like "any invalid OTP state."